src/main/java/de/adesso/kicker/season/persistence/SeasonStatistic.java

```diff
@@ -0,0 +1,34 @@
+package de.adesso.kicker.season.persistence;
+
+import de.adesso.kicker.user.persistence.User;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+public class SeasonStatistic {
```

Comment:
Not sure if Both classes are neccessary. 
If we would only provide the `SeasonTrackedStatistic`s that would contain the purpose of this class, if a last object would be created right before resetting the season.

The other (not so nice) way would be to only save the 'SeasonStatistic' to provide only the really relevant information.

The only way both classes makes sense would be if we would provide the option to revisit profiles for older seasons.